### DISCUSSION TITLE
Feature Request: Requested Docs

### DISCUSSION THREAD

**Discussion by matbrgz** [outline/outline#1466] [General]
![image](https://user-images.githubusercontent.com/23284276/90511606-1af3fa00-e133-11ea-9b31-654c27dc7b08.png)

**Comment by pclaure** [comment]
@MatheusRV .. I'm running this task, any suggestions.

**Reply by matbrgz** [reply]
@tommoor pclaure is working if us could you help here?

**Reply by tommoor** [reply]
I'll have to put together a spec document and put some thought into the design before code starts, definitely worth it's own database table though and probably an item in the sidebar. I'm supportive of this feature
